lib/assert.js

```diff
@@ -341,6 +358,198 @@ assert.notStrictEqual = function notStrictEqual(actual, expected, message) {
   }
 };
 
+function isPlainObject(obj) {
+  if (typeof obj !== 'object' || obj === null) {
+    return false;
+  }
+  const proto = ObjectGetPrototypeOf(obj);
+  return proto === ObjectPrototype || proto === null || ObjectPrototypeToString(obj) === '[object Object]';
+}
```

Comment:
ah I get it now!  
so you prefer to outline every single case where it is NOT an `object` instead of when it IS an `object`, I am not sure I like this approach tbh 😃

Reply:
@aduh95 ahhh now I see what you mean 😃 sorry for the confusion. I thought the list of cases would have gone longer, reading the proposal unreadable 😄